README.md

```diff
@@ -61,23 +61,88 @@ See
 The game depends on some tools and libraries to be present, before you can start
 building it. Here's a list of them:
 
-- A compiler that supports C++17, as that is the standard we follow now.
-- SCons build tool
-  http://www.scons.org/
 - Python3 (for build scripts)
 - SDL 2
   http://www.libsdl.org
 - SDL 2 TTF
   https://wiki.libsdl.org/SDL2_ttf/FrontPage
 - SDL_mixer 2
   http://www.libsdl.org/projects/SDL_mixer/
-- Git (to fetch the source from repository)
-  https://git-scm.com/
 
 Thanks to all the authors of these helpful libraries that made our development
 easy and straightforward.
 
-To build the game and run as normal user do the following:
+We are in the process of deprecating the [Scons](https://scons.org/) build
+system in favor of [Meson](https://mesonbuild.com/). You still may use either
+one (or both). Note that our meson build doesn't yet support Windows.
+
+## Meson
+
+Instructions for installing `meson` are on their website (see link above).
+
+Install the `physfs` library (required by NetPanzer).
```

Comment:
1. why doesn't the static link to the lib in the repo work?
2. until then let's just list this above with other dependencies

Reply:
I didn't add the code to build physfs to the meson.build, just a rule to dynamically link.

Debian patches netpanzer so it links to phyfs and lua. That needs to be done here on the upstream. No reason to build them when they're already widely available through Linux repos.

For windows, we'll be using the meson wrap system. https://github.com/netpanzer/netpanzer/pull/46#discussion_r1442161364 unless the deps can be installed with mingw or cygwin or something.

Reply:
> No reason to build them when they're already widely available through Linux repos.
Except for pinning a specific version.

Reply:
>     2. until then let's just list this above with other dependencies

Are you sure that won't be confusing for Windows devs building with scons, since it's built automatically?

Noting which build systems require which specific deps is only temporary until meson can build on Windows, but if you want, I can just list all the deps in once place like you asked.